### PR TITLE
Sticky columns don't support filtering, as it's done through the search

### PR DESCRIPTION
**Pull request by phndiaye**

### CODE REVIEW

**Comment by Hegazy360** [comment]
LGTM

File: app/templates/components/hyper-table/cell-renderers/image.hbs [hunk 1]
```diff
@@ -5,7 +5,9 @@
 {{#if (gt column.labels.length 0)}}
   <div>
     {{#each column.labels as |label|}}
-      <span>{{get item label}}</span>
+      <span data-toggle="tooltip" data-title={{get item label}}>
```
Comment (tgallice) [inline app/templates/components/hyper-table/cell-renderers/image.hbs]: there is a real gain to add the same value as `data-title` ?

Comment (phndiaye) [reply app/templates/components/hyper-table/cell-renderers/image.hbs]: it's because the column can be truncated with ellipsis

Comment (tgallice) [reply app/templates/components/hyper-table/cell-renderers/image.hbs]: good point :+1:

File: app/templates/components/hyper-table.hbs [hunk 2]
```diff
@@ -78,7 +78,8 @@
           {{/if}}
 
           <div class="hypertable__column">
-            {{hyper-table/cell column=column header=true manager=manager}}
+            {{hyper-table/cell
+              column=column header=true manager=manager sticky=true}}
```
Comment (tgallice) [inline app/templates/components/hyper-table.hbs]: Just to understand, the `sticky` property is only apply to first column and can not be apply to other ones ?

Comment (phndiaye) [reply app/templates/components/hyper-table.hbs]: yep ! but i realise I could pass something lik `allowsFiltering` on the columns too. that way we could use it on all columns potentially

**Pull request merged**